meilisearch/client.py

```diff
@@ -161,6 +161,25 @@ def health(self):
         """
         return self.http.get(self.config.paths.health)
 
+    def is_healthy(self):
+        """Get health of the MeiliSearch server.
+
+        `200` HTTP status response when MeiliSearch is healthy.
+
+        Return
+        ------
+        health: True | False
+        """
+        try:
+            self.http.get(self.config.paths.health)
```

Comment:
You should call the `health()` method here to avoid code duplication

Reply:
Done!